server/plugin/api.go

```diff
@@ -88,41 +122,51 @@ func (p *MatterpollPlugin) handleVote(w http.ResponseWriter, r *http.Request) {
 	}
 	userID := request.UserId
 
+	user, appErr := p.API.GetUser(userID)
+	if appErr != nil {
+		p.API.LogError("failed to fetch user")
+		response.EphemeralText = commandErrorGeneric.Other
+		writePostActionIntegrationResponse(w, response)
+		return
+	}
+	localizer := i18n.NewLocalizer(p.bundle, user.Locale)
+
 	poll, err := p.Store.Poll().Get(pollID)
 	if err != nil {
-		response.EphemeralText = commandGenericError
+		response.EphemeralText = localizer.MustLocalize(&i18n.LocalizeConfig{DefaultMessage: commandErrorGeneric})
```

Comment:
Might be worth to move ` localizer.MustLocalize(&i18n.LocalizeConfig{}` into a separate method.

Reply:
I think so too.   Would also shorten up the above prettier issue.

Reply:
Let's resolve the other conversation first and see if this problem still exists.